synth/validator/miner_data_handler.py

```diff
@@ -227,21 +327,45 @@ def get_miner_scores(self, scored_time_str: str, cutoff_days: int):
             return df
         except Exception as e:
             bt.logging.error(f"in get_miner_scores (got an exception): {e}")
+            traceback.print_exc(file=sys.stderr)
             return pd.DataFrame()
 
+    def populate_miner_uid_in_miner_rewards(
+        self, miner_rewards_data: list[dict]
+    ):
+        try:
+            with self.engine.connect() as connection:
+                with connection.begin():  # Begin a transaction
+                    miner_uid_map = self.get_miner_ids_map(connection)
+        except Exception as e:
+            bt.logging.error(
+                f"in update_miner_rewards (got an exception): {e}"
+            )
+            traceback.print_exc(file=sys.stderr)
+            return None
+
+        for row in miner_rewards_data:
+            miner_id = row["miner_id"]
+            # miner_uid column is deprecated
+            row["miner_uid"] = (
```

Comment:
didn't understand this part... why do you still want to insert the miner_uid if you deprecated it?

Reply:
I want to keep the miner_uid column in the database for some days to be sure it works well, and then we will be able to drop those column. For this migration, I don't want to do destructive action, to not risque to lose when prediction or score belongs to what uid.

Reply:
yep, that's reasonable, thanks for the clarification